tower/tests/builder.rs

```diff
@@ -1,3 +1,5 @@
+#![cfg(feature = "broken")]
```

Comment:
I don't see this one documented or used anywhere else. What's up?

Reply:
Yeah, I need buffer to get merged before I can renable this

Reply:
I meant to open this as a draft.

Reply:
Gotcha.

Reply:
If you rebase this atop of the buffer changes, I'll be able to approve?